Wangscape/noise/module/Wrapper.cpp

```diff
@@ -0,0 +1,67 @@
+#include "Wrapper.h"
+
+namespace noise
+{
+namespace module
+{
+
+template<> void Wrapper<ModuleGroup>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Wrapper<Billow>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Wrapper<Perlin>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Wrapper<RidgedMulti>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Wrapper<Turbulence>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Wrapper<Voronoi>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
```

Comment:
It's possible to use template magic to write one specialisation for all `M`with `SetSeed`, but for now the concise and readable version of that template isn't compatible with my compiler. And the number of modules with `SetSeed` isn't too large.